README.md

```diff
@@ -10,150 +10,148 @@
+## Onboarding Your Users
 
-```
-"PluginSettings": {
-    ...
-    "Plugins": {
-        "github": {
-            "encryptionkey": "<your encryption key, from step 5 above>",
-            "githuboauthclientid": "<your oauth client id, from step 2 above>",
-            "githuboauthclientsecret": "<your oauth client secret, from step 2 above>",
-            "githuborg": "<your github org>",
-            "webhooksecret": "<your webhook secret, from step 3 above>"
-        },
-    ...
-    "PluginStates": {
-        ...
-        "github": {
-            "Enable": true
-        },
-        ...
-    }
-},
-```
+When you’ve tested the plugin and confirmed it’s working, notify your team so they can connect their GitHub account to Mattermost and get started. Copy and paste the text below, edit it to suit your requirements, and send it out.
 
-## Development
+> Hi team, 
 
-This plugin contains both a server and web app portion.
+> We've set up the Mattermost GitHub plugin, so you can get notifications from GitHub in Mattermost. To get started, run the `/github connect` slash command from any channel within Mattermost to connect your Mattermost account with GitHub. Then, take a look at the [slash commands](#slash-commands) section for details about how to use the plugin.
 
-- Use `make dist` to build distributions of the plugin that you can upload to a Mattermost server.
-- Use `make check-style` to check the style.
-- Use `make deploy` to deploy the plugin to your local server. 
+## Slash Commands
 
-Before running `make deploy` you need to set a few environment variables:
+* __Autocomplete slash commands__ - Explore all the available slash commands by typing `/` in the text input box - the autocomplete suggestions help by providing a format example in black text and a short description of the slash command in grey text. Visit the [executing commands](https://docs.mattermost.com/help/messaging/executing-commands.html) documentation for more details.
+* __Subscribe to a respository__ - Use `/github subscribe` to subscribe a Mattermost channel to receive notifications for new pull requests, issues, branch creation, and more in a GitHub repository.
 
-```
-export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
-export MM_ADMIN_USERNAME=admin
-export MM_ADMIN_PASSWORD=password
-```
+   - For instance, to post notifications for issues, issue comments, and pull requests matching the label `Help Wanted` from `mattermost/mattermost-server`, use:
+   ```
+   /github subscribe mattermost/mattermost-server issues,pulls,issue_comments,label:"Help Wanted"
+   ```
+  - The following flags are supported:
+   - `--exclude-org-member`: events triggered by organization members will not be delivered. It will be locked to the organization provided in the plugin configuration and it will only work for users whose membership is public. Note that organization members and collaborators are not the same.
```

Comment:
```suggestion
     - `--exclude-org-member`: events triggered by organization members will not be delivered. It will be locked to the organization provided in the plugin configuration and it will only work for users whose membership is public. Note that organization members and collaborators are not the same.
```